packages/mobile/e2e/scripts/fund-e2e-accounts.ts

```diff
@@ -14,11 +14,16 @@ const kit = ContractKit.newKitFromWeb3(web3)
 
 // Throw error if balance is low
 const balanceError = async (address = valoraE2ETestWallet, minBalance = 10) => {
-  let balanceObject = await getBalance(address)
-  for (const balance in balanceObject) {
-    if (balanceObject[balance] < minBalance) {
-      throw new Error(`Balance of ${address} is below ${minBalance}`)
+  try {
+    let balanceObject = await getBalance(address)
+    for (const balance in balanceObject) {
+      if (balanceObject[balance] < minBalance) {
+        throw new Error(`Balance of ${address} is below ${minBalance}`)
+      }
     }
+  } catch (err) {
+    console.log(err)
+    process.exit(1)
   }
```

Comment:
Though this works for this specific function, there's a better alternative which would exit with a non-zero status code on any unhandled promise rejection:

```js
process.on("unhandledRejection", (error) => {
  throw error;
});
```

or alternatively call the script with:
`node --unhandled-rejections=throw some-script.js`

`--unhandled-rejections=throw` is now the default since Node 15, see https://nodejs.org/api/cli.html#--unhandled-rejectionsmode